### PR TITLE
dev: adds update script for server

### PR DESCRIPTION
**Pull request by jasperalani**
https://discord.com/channels/890222432605057044/1190240450494353430

### CODE REVIEW

**Comment by jasperalani** [comment]
the shell script needs to be tested

**Review by Yarden-zamir — CHANGES_REQUESTED** [review]
The kubejs directory not found sounds like an error but is not necasserally so better to rephrase it to or remove the else part

**Review by Yarden-zamir — APPROVED** [review]
Checked the shell script, works well
